IdentityCore/src/util/NSDictionary+MSIDExtensions.h

```diff
@@ -23,6 +23,7 @@
 
 @interface NSDictionary (MSIDExtensions)
 
++ (NSDictionary *)msidDictionaryFromString:(NSString *)string;
```

Comment:
nit: can we call it something that represents what kind of string it converts from (e.g. FromQueryString)?